.werft/util/certs.ts

```diff
@@ -33,7 +33,46 @@ export async function issueCertificate(werft: Werft, params: IssueCertificatePar
 
     exec(`echo "Domain: ${params.domain}, Subdomains: ${subdomains}"`, { slice: shellOpts.slice });
     validateSubdomains(werft, shellOpts.slice, params.domain, subdomains);
-    createCertificateResource(werft, shellOpts, params, subdomains);
+
+    const maxAttempts = 5
+    const timeout = "150s"
```

Comment:
Tiny nit: As this is only used in checkCertReadiness I'd just move it there. Then it's also more clear why it's a string